sda-commons-server-opentelemetry/src/main/java/org/sdase/commons/server/opentelemetry/decorators/HeadersUtils.java

```diff
@@ -0,0 +1,91 @@
+package org.sdase.commons.server.opentelemetry.decorators;
+
+import static javax.ws.rs.core.HttpHeaders.AUTHORIZATION;
+import static javax.ws.rs.core.HttpHeaders.COOKIE;
+import static javax.ws.rs.core.HttpHeaders.SET_COOKIE;
+import static org.apache.commons.lang3.StringUtils.join;
+
+import io.opentelemetry.api.common.AttributeKey;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.ws.rs.core.MultivaluedHashMap;
+import javax.ws.rs.core.MultivaluedMap;
+
+public class HeadersUtils {
+
+  private HeadersUtils() {
+    // prevent instances
+  }
+
+  /**
+   * Please use {@link HeadersUtils#convertHeadersToString(MultivaluedMap)} to generate values for
+   * this tag.
+   */
+  public static final AttributeKey<List<String>> HTTP_REQUEST_HEADERS =
+      AttributeKey.stringArrayKey("http.request_headers");
+
+  /**
+   * Please use {@link HeadersUtils#convertHeadersToString(MultivaluedMap)} to generate values for
+   * this tag.
+   */
+  public static final AttributeKey<List<String>> HTTP_RESPONSE_HEADERS =
+      AttributeKey.stringArrayKey("http.response_headers");
+
+  /**
+   * Convert a given {@link MultivaluedMap} with {@link String} keys to the format [key0 = 'value0',
+   * 'value1']; [key1 = 'value2']; ...
```

Comment:
Am I confused because of the linebreak or is there a mistake in the example?

Reply:
extracted a separate method and updated the example